hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/fs/TestWebHdfsFileContextMainOperations.java

```diff
@@ -63,7 +64,7 @@ public Path getDefaultWorkingDirectory() {
 
   @Override
   protected FileContextTestHelper createFileContextHelper() {
-    return new FileContextTestHelper("/tmp/TestWebHdfsFileContextMainOperations");
+    return new FileContextTestHelper();
```

Comment:
I found some other occurrences of `/tmp/xxx`, which is used as the root dir of `FileContextTestHelper`. Would you replace them? I'm thinking the following is fine:
```
return new FileContextTestHelper(GenericTestUtils.getTempPath("TestWebHdfsFileContextMainOperations"));
```
I'm okay if the replace is done in separate jira(s).